server/routerlicious/packages/routerlicious-base/src/test/nexus/io.spec.ts

```diff
@@ -58,6 +58,7 @@ import {
 	isTokenRevokedError,
 	type IRevokedTokenChecker,
 } from "@fluidframework/server-services-core/dist/tokenRevocationManager";
+import { isSentSignalMessage } from "@fluidframework/server-lambdas/dist/nexus/utils";
```

Comment:
From a clean testing standpoint, we should not use the logic of some product code that might be in error in the testing flow. The effectiveness of the tests shouldn't be changed by changing production code. It would be okay to have a similar helper duplicated here. Or you have to test that helper before using it - I do not recommend testing the helper directly. A duplicated helper can use what it knows about local test cases to have it be simpler.

Reply:
Created a duplicate helper